cadc-vos-server/src/main/java/org/opencadc/vospace/server/PathResolver.java

```diff
@@ -182,8 +186,8 @@ public Node getNode(String nodePath) throws Exception {
                 ret = child;
             }
         }
-        
-        log.debug("return node: " + Utils.getPath(ret));
+
+        log.debug("return node: " + (( ret != null) ? Utils.getPath(ret) : null));
```

Comment:
Is there another way to get here with ret==null? There are already immediate `return null;` up at line 146 and 179

Reply:
Sometimes when it comes out of lane 85. I've already run into it. Although now with the alternative implementation in `CavernURLGenerator` might not be the case, it is still possible.